docs/EN_US/ECLStandardLibraryReference/SLR-Mods/getElapsedMs.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect1 PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<sect1 id="getElapsedMs">
+  <title>getElapsedMs</title>
+
+  <para><emphasis>result</emphasis> <emphasis role="bold"> :=
+  STD.System.Log.getElapsedMs<indexterm>
+      <primary>STD.System.Log.getElapsedMs</primary>
+    </indexterm> <indexterm>
+      <primary>System.Log.getElapsedMs</primary>
+    </indexterm> <indexterm>
+      <primary>Log.getElapsedMs</primary>
+    </indexterm> <indexterm>
+      <primary>getElapsedMs</primary>
+    </indexterm></emphasis><emphasis></emphasis><emphasis role="bold">
+  ();</emphasis></para>
+
+  <informaltable colsep="1" frame="all" rowsep="1">
+    <tgroup cols="2">
+      <colspec colwidth="80.50pt" />
+
+      <colspec />
+
+      <tbody>
+        <row>
+          <entry>Return:</entry>
+
+          <entry>getElapsedMs returns returns the elapsed time in
+          milliseconds.</entry>
+        </row>
+      </tbody>
+    </tgroup>
+  </informaltable>
+
+  <para>The <emphasis role="bold">getElapsedMs </emphasis>function returns the
+  current elapsed query time (in ms) in Roxie.</para>
```

Comment:
You may want to mention that this is the elapsed time when STD.System.Log.getElapsedMs() is called.  Because ECL is a declarative language that can sometimes be confusing.  ECL code is not executed in sequence so you have to be careful when trying to get the elapsed time for a particular point.  @ghalliday may have better wording for that topic.